database/repositories/settings.py

```diff
@@ -7,3 +7,10 @@ def at_key(self, key: str) -> str:
         value = session.query(self.model).filter_by(key=key).first().value
         session.close()
         return value
+
+    def load_from_yaml(self, data):
+        session = self.sessionmaker()
+        for key, value in data.items():
+            session.add(self.model(key=key, value=value))
+        session.commit()
+        session.close()
```

Comment:
dużo duplikatów z tą funkcją `load_from_yaml` w osobnych repo, wydziel to do jednej